cg/apps/mip/base.py

```diff
@@ -38,13 +38,14 @@
 class MipAPI:
     """ Group MIP specific functionality """
 
-    def __init__(self, script: str, pipeline: str, conda_env: str):
+    def __init__(self, script: str, pipeline: str, conda_env: str, root: str, **kwargs):
```

Comment:
Could you comment in a docstring why root is there and `**kwargs`. Looks like kwargs is passed but not used

Reply:
You're right, root isn't supposed to be there. It was an artifact leftover from something I tried earlier... 
Kwargs are there for seamless inheritance. So instead of making connector class to combine AnalysisAPI and and MipAPI, MipAPI is made flexible. Is this bad practice?

Reply:
@moonso this is last one to be addressed before all feedback is incorporated. Would be good to know if this is a terrible design decision before proceeding!